tests/test_oauth2.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+import requests
+import responses
+
+logging.basicConfig(level=logging.DEBUG)
+
+token = None
+
+
+# try:
+#     import urlparse
+# except ImportError:
+#     import urllib.parse as urlparse
+
+
+def get_token():
+    return token
+
+
+def set_token(tkn):
+    global token
+    token = tkn
+
+
+@responses.activate
+def test_oauth2(client):
+    client_id = "app_TuUwqdU76H8kFaB5hsrVGdMp"
```

Comment:
Please add a docstring that explains what this test does. It looks like an integration test that tests the authentication flow, but I'm not sure.

Reply:
Added docstring: `"""Test oauth2 integration"""`. @kingel any hints to improve this?

Reply:
How about "Let the client perform a full authorization workflow."?

Reply:
Dropped